tests/test_enki.py

```diff
@@ -505,6 +505,19 @@ def test_extract_bibliographic(self):
         eq_(Representation.EPUB_MEDIA_TYPE, format.content_type)
         eq_(DeliveryMechanism.ADOBE_DRM, format.drm_scheme)
 
+    def test_extract_bibliographic_pdf(self):
+        """Test the ability to distingush between PDF and EPUB results"""
+        data = json.loads(self.get_data("pdf_document_entry.json"))
+        parser = BibliographicParser()
+        m = parser.extract_bibliographic(data['result'])
+        assert isinstance(m, Metadata)
+
+        # The book is available as a non-ACS PDF.
+        circulation = m.circulation
+        assert isinstance(circulation, CirculationData)
+        [format] = circulation.formats
+        eq_(Representation.PDF_MEDIA_TYPE, format.content_type)
+        eq_(DeliveryMechanism.NO_DRM, format.drm_scheme)
```

Comment:
OK, that's not that I was expecting, but looking at pdf_document_entry.json it is in fact a DRM-free PDF. So line 515 is wrong. Just change that comment and you'll be good to go.

Reply:
I did, It now says "The book is available as a non-ACS PDF" or should it be worded differently?

Reply:
No, that's great.